borgmatic/config/schema.yaml

```diff
@@ -931,6 +931,31 @@ properties:
                     mysqldump/mysql commands (from either MySQL or MariaDB). See
                     https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html or
                     https://mariadb.com/kb/en/library/mysqldump/ for details.
+            sqlite_databases:
+                type: array
+                items:
+                    type: object
+                    required: ['path','name']
+                    additionalProperties: false
+                    properties:
+                        path:
+                            type: string
+                            description: |
+                                Path to the SQLite database file to dump. If
+                                relative, it is relative to the current working
+                                directory. If absolute, it is relative to the
+                                root of the filesystem. Note that using this
+                                database hook implicitly enables both
+                                read_special and one_file_system (see above) to
+                                support dump and restore streaming.
+                            example: /var/lib/sqlite/users.db
+                        name:
+                            type: string
+                            description: |
+                                This is used to tag the database dump file with
+                                a name. It is not used to identify the database
+                                file itself.
```

Comment:
This second sentence might be clearer if it was instead something like: "It is not the path to the database file itself."